common/config/rush/build-cache.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/build-cache.schema.json",
+  "buildCacheEnabled": false,
+  "cacheProvider": "azure-blob-storage",
+  "azureBlobStorageConfiguration": {
+    "storageAccountName": "itwintestrushcache",
```

Comment:
@ben-polinsky let's follow up on the location and name of this storage account. putting it here so we don't forget.

Reply:
updated